simpeg_drivers/components/factories/directives_factory.py

```diff
@@ -412,12 +413,13 @@ def assemble_keyword_arguments(
         active_cells_map = maps.InjectActiveCells(
             inversion_object.mesh, active_cells, np.nan
         )
+
         sorting = inversion_object.permutation
         kwargs = {
             "label": "model",
             "association": "CEll",
             "dmisfit": global_misfit,
-            "transforms": [active_cells_map],
+            "transforms": [active_cells_map, sqrt],
```

Comment:
Let's add volume now I guess. So you don't have to it later.

Reply:
done.